garden_ai/garden.py

```diff
@@ -79,3 +81,65 @@ def _authenticate(self):
             on_refresh=self.auth_key_store.on_refresh,
         )
         return authorizer
+
+    def create_garden(self, **kwargs):
```

Comment:
My vision was to make the required fields named parameters in this constructor so it's clear in your IDE exactly what is the minimum for creating a Garden.

Reply:
I like that idea -- we could also just expose the `Garden` constructor to the user, because my impression is that pydantic makes a point of playing nicely with IDEs/etc

Reply:
Oh, but that would be providing completion suggestions for all the fields, not *only* the required fields, which I think is what you were getting at

Reply:
It would be something like `create_garden(title: str, authors: list[str], **kwargs)` 

So the mandatory fields would be obvious, and the rest would be keywords